app/routes/events/components/EventEditor/renderPools.tsx

```diff
@@ -15,11 +16,6 @@ type poolProps = {
   eventStatusType: EventStatusType;
 };
 
-const highWarning = (value) =>
-  value && value >= 500
-    ? 'Åpent arrangement gir uendelig plasser 😁'
-    : undefined;
-
```

Comment:
Why remove this?

Reply:
Warning are currently not supported with react-final-form

Reply:
Yeah, I figure we might re-add warnings where we want to at some point, but for now it's just dead code throwing console errors - so I'd rather remove it